todo_app/data/item.py

```diff
@@ -0,0 +1,55 @@
+class Item:
+    
+    # def __init__(self, id, title, description, status):
```

Comment:
We might as well delete this now it's been replaced

Reply:
This is now deleted